javascript/javascript1/week3/SeriesDurationOfMyLife.js

```diff
@@ -0,0 +1,42 @@
+const seriesDurations = [
+    {
+        title: "Game of thrones",
+        days: 3,
+        hours: 1,
+        minutes: 0,
+    },
+    {
+        title: "Sopranos",
+        days: 3,
+        hours: 14,
+        minutes: 0,
+    },
+    {
+        title: "The Wire",
+        days: 2,
+        hours: 12,
+        minutes: 0,
+    },
+];
+
+const lifeSpan = 80 * 365 * 24 * 60
+let totalSeriesTime = 0
+
+function logOutSeriesText(object) {
```

Comment:
object is a very general parameter name. Can you come up with something that explains the meaning of the data?

Reply:
okay .. changed object into 'seriesDurations'